handler.go

```diff
@@ -66,19 +89,47 @@ func (h *RollbarHandler) Enabled(_ context.Context, level slog.Level) bool {
 
 func (h *RollbarHandler) Handle(ctx context.Context, record slog.Record) error {
 	fromContext := slogcommon.ContextExtractor(ctx, h.option.AttrFromContext)
-	extra := h.option.Converter(h.option.AddSource, h.option.ReplaceAttr, append(h.attrs, fromContext...), h.groups, &record)
+	extra := h.option.Converter(h.option.AddSource, h.option.ReplaceAttr,
+		append(h.attrs, fromContext...), h.groups, &record)
 	level := LogLevels[record.Level]
 
 	ctx, cancel := context.WithTimeout(context.Background(), h.option.Timeout)
 	defer cancel()
 
-	// if level == rollbar.ERR || level == rollbar.CRIT {
-	// 	skip := framesToSkip(2)
-	// 	h.option.Client.ErrorWithStackSkipWithExtrasAndContext(ctx, rollbar.ERR, err, skip, extra)
-	//  return nil
-	// }
-
-	h.option.Client.MessageWithExtrasAndContext(ctx, level, record.Message, extra)
+	// extract error and request from slog.Record
+	var r *http.Request
+	var err error
+
+	record.Attrs(func(a slog.Attr) bool {
+		if err != nil && r != nil {
+			return false
+		}
+
+		switch v := a.Value.Any().(type) {
+		case *http.Request:
+			// Pass the request to rollbar for additinoal context
+			r = v
+		case error:
+			// Keep the original message, but report as an error to Rollbar
+			// This enables stack tracing if the error has it.
+			err = wrappedError{msg: record.Message, error: v}
```

Comment:
why not simply `fmt.Errorf("%s: %w", record.Message, v)` ? 🤔

Reply:
To avoid changing the original logged message, as this is not a new error, and I think that would be unexpected.

If I want to log `this happened`, I would be surprised if I got `this happened: and here is an error we added to it`.